about.html

```diff
@@ -12,53 +12,72 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+  
+  <nav class="home-navigation">
+    <a href="index.html"><img class="lambda-black" src="img/lambda-black.png" alt="Lambda logo"></a>
```

Comment:
you didn't have to make this img an anchor though it is nice to be able to use to go back to the main page